Assets/FantasyEnvironments/Environments/Scenes/DemoJoeyG2.unity

```diff
@@ -0,0 +1,2521 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
+OcclusionCullingSettings:
+  m_ObjectHideFlags: 0
```

Comment:
Reminder to move this file into the Assets/Scenes folder instead of hidden in FantasyEnvironments

Reply:
I moved it but there was an old copy so I renamed it newDemoScene